### DISCUSSION TITLE
Enable zoom in on modals? (Non-mobile devices)

### DISCUSSION THREAD

**Discussion by mariobd38** [mantinedev/mantine#6074] [Q&A]
Hi,
I'm just starting out with Mantine and making use of the Modal component for my project. However, seems it is not possible to pinch to zoom when the modal is opened, except for mobile devices, as presented here: https://help.mantine.dev/q/pinch-to-zoom-modal

Any way I can enable 'pinch to zoom' while disabling/locking scroll as long as the modal is opened?

Thanks!

**Comment by rtivital** [comment]
No, if you need this feature, you are disable a built in scroll lock feature and implement scroll lock on your side.
